docs/apidiffs/current_vs_latest/opentelemetry-exporter-zipkin.txt

```diff
@@ -1,2 +1,12 @@
 Comparing source compatibility of  against 
-No changes.
\ No newline at end of file
++++  NEW CLASS: PUBLIC(+) FINAL(+) io.opentelemetry.exporter.zipkin.OtelToZipkinSpanTransformer  (not serializable)
+	+++  CLASS FILE FORMAT VERSION: 52.0 <- n.a.
+	+++  NEW INTERFACE: java.util.function.Function
+	+++  NEW SUPERCLASS: java.lang.Object
+	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.util.logging.Logger logger
+	+++  NEW CONSTRUCTOR: PUBLIC(+) OtelToZipkinSpanTransformer()
+	+++  NEW CONSTRUCTOR: PUBLIC(+) OtelToZipkinSpanTransformer(java.net.InetAddress)
```

Comment:
is this constructor only for testing, or is it intended to be public?

Reply:
Intended to be public. The Splunk Android SDK will need to call this with `null`. In the longer term, we may allow passing a `Supplier` as mentioned in #4676.

Reply:
I wonder if we should not defer this to the longer term... Then we'd have both APIs, and that might be more confusing. Any reason we don't want to change this PR to have a supplier?

Reply:
Oh sure, that sounds good. Lemme give it a tweak.